modules/ossm-rn-known-issues.adoc

```diff
@@ -46,6 +44,10 @@ Workaround: Use the `techPreview` `jwksResolverExtraRootCA` field to enable addi
 +
 After enabling the `spec.security.controlPlane.mtls` setting in the SMCP, the Kiali console displays the following error message `No subsets defined`.
 
+* https://issues.redhat.com/browse/OSSM-1505[OSSM-1505] `ServiceMeshExtension` failed to be ready. If you are using Service Mesh with `ServiceMeshExtension` on OpenShift Container Platform 4.11 then the `ServiceMeshExtension` resource never becomes ready. If you inspect the issue using `oc describe ServiceMeshExtension` you will see the following error:  `stderr: Error creating mount namespace before pivot: function not implemented`.
```

Comment:
- Perhaps it should be "`ServiceMeshExtension` fails to be ready."? I'm not a native English speaker, but to me, it reads better if put in present, not in past tense.
- Suggestion: Drop the "If you are using Service Mesh" part. If users are reading this, they are likely users of Service Mesh. What about: If you are using the `ServiceMeshExtension` resource on OpenShift Container Platform 4.11 then the you will notice that it never becomes ready."

Reply:
@jwendell good suggestion you made! I updated the title of the Jira and the release notes content now matches.